flash/core/data/data_source.py

```diff
@@ -543,3 +554,256 @@ def _get_classes(self, data):
             classes = data.distinct(label_path)
 
         return classes
+
+
+class LabelStudioDataset(Dataset):
```

Comment:
Is this used anywhere? Looks like it can be removed

Reply:
Fixed